include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -1480,14 +1497,15 @@ __pattern_partial_sort_copy(_ExecutionPolicy&& __exec, _InIterator __first, _InI
     {
         // If our output buffer is larger than the input buffer, simply copy elements to the output and use
         // full sort on them.
-        auto __out_end = __pattern_walk2</*_IsSync=*/::std::false_type>(
-            __par_backend_hetero::make_wrapped_policy<__initial_copy_1>(__exec), __first, __last, __out_first,
-            __brick_copy<_ExecutionPolicy>{}, ::std::true_type{}, ::std::true_type{});
+        auto __out_end = __pattern_walk2<_BackendTag, /*_IsSync=*/::std::false_type>(
```

Comment:
why do you need having `_BackendTag` as template argument? If it's because of `_IsSync` then we need to swap template parameters order. So far, looks redundant

Reply:
Fixed